src/github.com/matrix-org/dendrite/clientapi/common/common.go

```diff
@@ -8,6 +8,15 @@ import (
 	"github.com/matrix-org/util"
 )
 
+// StateTuple is the tuple of an event type and an event state_key, typically used as a key
+// in maps.
+type StateTuple struct {
```

Comment:
This is called StateKeyTuple in roomserver. I dunno if we want to move this into somewhere that can be shared between them.